docs/examples/getting_started.md

```diff
@@ -98,30 +114,28 @@ Optional Arguments
 Follow [here](./../cmf_client/cmf_client.md#cmf-init) for more details.
 
 ## Project execution
-To execute the example pipeline, run the 
-[test_script.sh](../../examples/example-get-started/test_script.sh) 
-file (before that, study the contents of that file). Basically, that script will run a sequence of steps
-common for a typical machine learning project - getting raw data, converting it into machine learning train/test splits,
-training and testing a model. The execution of these steps (and parent pipeline) will be recorded by the CMF.
+To execute the example pipeline, run the
+[test_script.sh](../../examples/example-get-started/test_script.sh) file. Basically, that script will run the sequence of steps
+described in the Overview section above.
 ```shell
 # Run the example pipeline
 sh ./test_script.sh
 ```
 
-This script will run the pipeline and will store its metadata in a sqlite file named mlmd. Verify that all stages are 
-done using `git log` command. You should see commits corresponding to the artifacts that were created.
+This script will run the pipeline and will store its metadata in a sqlite file named mlmd. Verify that all were completed by running the `git log` command. Each created artifact should be associated with its own commit.
 
-Under normal conditions, the next steps would be to: (1) execute the `cmf artifact push` command to push the artifacts to the central artifact repository and (2) execute the `cmf metadata push` command to track the metadata of the generated artifacts on a common [cmf server](./../cmf_server/cmf-server.md).
+In typical produciton environments, the next steps would be to: (1) execute the `cmf artifact push` command to push the artifacts to the central artifact repository and (2) execute the `cmf metadata push` command to integrate the metadata of the generated artifacts on a common [cmf server](./../cmf_server/cmf-server.md).
 
 Follow [here](./../cmf_client/cmf_client.md#cmf-init) for more details on `cmf artifact` and `cmf metadata` commands.
 
 
-## Query 
-The stored metadata can be explored using the query layer. Example Jupyter notebook 
+## Query
+The stored metadata can be explored using the query layer. An xample Jupyter notebook
 [Query_Tester-base_mlmd.ipynb](../../examples/example-get-started/Query_Tester-base_mlmd.ipynb) can be found in this directory.
 
-## Clean Up 
+## Clean Up
 Metadata is stored in sqlite file named "mlmd". To clean up, delete the "mlmd" file.
- 
+
 ## Steps to test dataslice
+AES: What does this mean? Is this germaine to this example or is it a more advanced feature that could be split into its own example (or a followon to this one)?
```

Comment:
This is a new concept introduced here with which I'm not familiar. A naive reading suggests that this would be to potentially test the machine-learning model with a new slice of data, however reading the python file, this seems to be referring to something directly related to CMF. Is this a more advanced topic or something that a beginner user should be aware of?